Fix benchmark queries that crash/hang with binary driver (#296)
ClickHouse/pg_clickhouse: CHANGELOG.md

    *   `percentile_disc(double)` → `quantileExactLow()`
    *   `percentile_disc(double[])` → `quantilesExactLow()`

### 🐞 Bug Fixes

> *   Fixed crashes when a single query ran more than one foreign scan on the
>     binary (native-protocol) driver at once — such as a correlated subquery or a
>     nested-loop join over foreign tables — which previously collided on one
>     shared connection. Each concurrently active scan now gets its own connection
>     ([#296]).

Reviewer: In general let's edit to sound less like the artificial text extruder.

```suggestion
*   Fixed crashes when a single query ran more than one foreign scan on the
    binary (native-protocol) driver at once, such as a correlated subquery or a
    nested-loop join over foreign tables, colliding in the single connection.
    Each concurrently active scan now gets its own connection ([#296]).
```
